### PR TITLE
[QA-319]: Updated IPVR scenario to generate single user ID for all 3 event types

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Updated IPVR scenarios to generate single user ID for all 3 event types

#### Changes:
- Removed the individual scenarios for F2F and IPV events.
- Created a separate function for sending messages for all events with the same user ID.

---

### Why?
Notify mock will trigger a notification if the user id is same for all the three events.

### CODE REVIEW

File: deploy/scripts/src/cri-kiwi/ipvr.ts [hunk 1]
```diff
@@ -152,31 +115,31 @@ const awsConfig = new AWSConfig({
...
-  const payload = generateAuthRequest()
+  const authPayload = generateAuthRequest()
 
   const authEventMessage = {
-    messageBody: JSON.stringify(payload)
+    messageBody: JSON.stringify(authPayload)
   }
 
   sqs.sendMessage(env.sqs_queue, authEventMessage.messageBody)
 }
```
Comment (Tom-Dann) [inline deploy/scripts/src/cri-kiwi/ipvr.ts:125]: Is this function no longer being used?

File: deploy/scripts/src/cri-kiwi/ipvr.ts [hunk 2]
```diff
@@ -152,31 +115,31 @@ const awsConfig = new AWSConfig({
...
-export function f2fEvent (): void {
-  const payload = generateF2FRequest()
+export function allEvents (): void {
+  const authPayload = generateAuthRequest()
+  const f2fPayload = generateF2FRequest()
+  const ipvPayload = generateIPVRequest()
 
+  const authEventMessage = {
+    messageBody: JSON.stringify(authPayload)
+  }
```
[the remark this answers is not in the document]
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/cri-kiwi/ipvr.ts:134]: Update in all the occurrences

File: deploy/scripts/src/cri-kiwi/ipvr.ts [hunk 3]
```diff
@@ -152,31 +115,24 @@ const awsConfig = new AWSConfig({
...
+  const ipvPayload = generateIPVRequest()
 
-  const f2fEventMessage = {
-    messageBody: JSON.stringify(payload)
-  }
-
-  sqs.sendMessage(env.sqs_queue, f2fEventMessage.messageBody)
-}
+  const authEventMessage = JSON.stringify(authPayload)
+  const f2fEventMessage = JSON.stringify(f2fPayload)
```
Comment (Tom-Dann) [inline deploy/scripts/src/cri-kiwi/ipvr.ts:132]: Could probably tidy up some of these blank new lines

File: deploy/scripts/src/cri-kiwi/ipvr.ts [hunk 4]
```diff
@@ -152,31 +115,24 @@ const awsConfig = new AWSConfig({
...
-
-  const ipvEventMessage = {
-    messageBody: JSON.stringify(payload)
-  }
+  const ipvEventMessage = JSON.stringify(ipvPayload)
 
-  sqs.sendMessage(env.sqs_queue, ipvEventMessage.messageBody)
+  sqs.sendMessage(env.sqs_queue, authEventMessage)
+  sqs.sendMessage(env.sqs_queue, f2fEventMessage)
+  sqs.sendMessage(env.sqs_queue, ipvEventMessage)
```
Comment (Tom-Dann) [inline deploy/scripts/src/cri-kiwi/ipvr.ts:137]:
```suggestion
  sqs.sendMessage(env.sqs_queue, JSON.stringify(authPayload))
  sqs.sendMessage(env.sqs_queue, JSON.stringify(f2fPayload))
  sqs.sendMessage(env.sqs_queue, JSON.stringify(ipvPayload))
```
There is some unnecessary variable declaration here

**Review by Tom-Dann — APPROVED** [review]

File: deploy/scripts/src/cri-kiwi/ipvr.ts (hunk 3 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/cri-kiwi/ipvr.ts]: Done

File: deploy/scripts/src/cri-kiwi/ipvr.ts (hunk 4 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/cri-kiwi/ipvr.ts]: Done. Tested this from local and the messages are getting generated successfully. Thanks.

**Pull request merged**